newIDE/app/src/VariablesList/VariablesListToolbar.js

```diff
@@ -115,25 +117,24 @@ const VariablesListToolbar = React.memo<Props>((props: Props) => {
           )}
         </Line>
       </Column>
-      <Column expand>
-        <SearchBar
-          value={props.searchText}
-          onRequestSearch={props.onChangeSearchText}
-          onChange={props.onChangeSearchText}
-          placeholder={t`Search variables`}
-        />
+      <Column expand noOverflowParent>
+        {props.isCompact ? (
+          <CompactSearchBar
+            value={props.searchText}
+            onChange={props.onChangeSearchText}
+            placeholder={t`Search variables`}
+          />
+        ) : (
+          <SearchBar
+            value={props.searchText}
+            onRequestSearch={props.onChangeSearchText}
+            onChange={props.onChangeSearchText}
+            placeholder={t`Search variables`}
+          />
+        )}
       </Column>
       <Column noMargin>
-        {props.isNarrow ? (
-          <IconButton
-            key="add-variable"
-            tooltip={t`Add variable`}
-            onClick={props.onAdd}
-            size="small"
-          >
-            <Add style={props.iconStyle} />
-          </IconButton>
-        ) : (
+        {props.isNarrow ? null : (
```

Comment:
remove completely this ability?

Reply:
This was a mistake, it should null only for isCompact. Nice catch